yarn.lock

```diff
@@ -4420,6 +4420,10 @@ output-file-sync@^1.1.0:
     mkdirp "^0.5.1"
     object-assign "^4.1.0"
 
+pagination-pager@2.4.2:
```

Comment:
I notice that you don't add any dependencies to package.json in this PR- was committing the change to your yarn.lock file intentional?